src/SprykerEco/Zed/Afterpay/Business/Payment/Mapper/QuoteToRequestTransfer.php

```diff
@@ -18,12 +18,10 @@
 use SprykerEco\Shared\Afterpay\AfterpayConstants;
 use SprykerEco\Zed\Afterpay\Dependency\Facade\AfterpayToMoneyInterface;
 
-
 //@todo extract shared behaviour between quote and order transfers if possible
 
 class QuoteToRequestTransfer implements QuoteToRequestTransferInterface
```

Comment:
Naming suggestion: `QuoteToRequestTransferMapper` — all `XxxTransfer` classes are supposed to be `transfer objects`